frontend/src/_ui/AppButton/AppButton.scss

```diff
@@ -1,7 +1,7 @@
 @import "../../_styles/designtheme.scss";
 
 .tj-base-btn {
-    box-sizing: border-box;
+    // box-sizing: border-box;
```

Comment:
Please make sure that this change doesn't affect any other components that use this class.